assets-and-images.md

```diff
@@ -207,6 +207,46 @@ or
 [`ImageCache`](https://docs.flutter.io/flutter/services/ImageCache-class.html),
 you'll also notice parameters related to scale.)
 
+### Asset images in package dependencies
+
+To load an image from a package dependency, the `package` argument must be provided to [`AssetImage`](https://docs.flutter.io/flutter/services/AssetImage-class.html).
```

Comment:
Maybe link `package` to https://flutter.io/using-packages/ which has a short intro to what packages are?